modules/Bio/EnsEMBL/Analysis/Hive/Config/StarScallopRnaseq.pm

```diff
@@ -0,0 +1,827 @@
+
+    'reference_db_name'   => $self->o('dna_db_name'),
+    'reference_db_server' => $self->o('dna_db_server'),
+    'reference_db_port'   => $self->o('dna_db_port'),
+
+    'star_rnaseq_for_layer_db_server' => $self->o('databases_server'),
+    'star_rnaseq_for_layer_db_port'   => $self->o('databases_port'),
+
+    'scallop_initial_db_server' => $self->o('databases_server'),
+    'scallop_initial_db_port'   => $self->o('databases_port'),
+
+    'scallop_blast_db_server' => $self->o('databases_server'),
+    'scallop_blast_db_port'   => $self->o('databases_port'),
+
+    # This is used for the ensembl_production and the ncbi_taxonomy databases
+    'ensembl_release' => $ENV{ENSEMBL_RELEASE},    # this is the current release version on staging to be able to get the correct database
+
+    databases_to_delete => ['reference_db'],
+
+########################
+# BLAST db paths
+########################
+    'base_blast_db_path' => $ENV{BLASTDB_DIR},
+
+######################################################
+#
+# Mostly constant settings
+#
+######################################################
+
+    genome_dumps => catdir( $self->o('output_path'), 'genome_dumps' ),
+    # This one is used in replacement of the dna table in the core db, so where analyses override slice->seq. Has simple headers with just the seq_region name. Also used by bwa in the RNA-seq analyses. Not masked
+    faidx_genome_file => catfile( $self->o('genome_dumps'), $self->o('species_name') . '_toplevel.fa' ),
+
+    'min_toplevel_slice_length' => 250,
+
+########################
+# Extra db settings
+########################
+
+    'num_tokens' => 10,
+
+########################
+# Executable paths
+########################
+    'star_path'    => '/homes/fergal/bin/STAR',
+    'scallop_path' => '/homes/fergal/bin/scallop',
+
+    'blast_type'             => 'ncbi',                                                                         # It can be 'ncbi', 'wu', or 'legacy_ncbi'
+    'uniprot_blast_exe_path' => catfile( $self->o('binary_base'), 'blastp' ),
+    'bwa_path'               => catfile( $self->o('software_base_path'), 'opt', 'bwa-051mt', 'bin', 'bwa' ),    #You may need to specify the full path to the bwa binary
+
+    # You have the choice between:
+    #  * using a csv file you already created
+    #  * using a study_accession like PRJEB19386
+    #  * using the taxon_id of your species
+    # 'rnaseq_summary_file' should always be set. If 'taxon_id' or 'study_accession' are not undef
+    # they will be used to retrieve the information from ENA and to create the csv file. In this case,
+    # 'file_columns' and 'summary_file_delimiter' should not be changed unless you know what you are doing
+    'study_accession'     => '',
+    'max_reads_per_split' => 2500000,      # This sets the number of reads to split the fastq files on
+    'max_total_reads'     => 200000000,    # This is the total number of reads to allow from a single, unsplit file
```

Comment:
This is not needed as we don't split files with star

Reply:
Removed.